src/components/Modal/LoginModal.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import Button from '../Button/Button';
+import Modal from './Modal';
+import './LoginModal.scss';
+
+const LoginModal = () => {
+  const [loginUserInfo, setLoginUserInfo] = useState({
+    name: '',
+    email: '',
+    password: '',
+  });
+
+  const emailPattern = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+  const emailIsValid = emailPattern.test(loginUserInfo.email);
+  const passwordIsValid = loginUserInfo.password.length >= 5;
```

Comment:
- is를 prefix로 붙여 주세요!
- loginUserInfo state를 구조분해할당해서 사용하면 더 사용하기 간편할 것 같습니다! 이 부분 말고도 반영해 주세요!
```suggestion
  const isEmailValid = emailPattern.test(email);
  const isPasswordValid = password.length >= 5;
```